pegsol/generator.rb

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/ruby
 
-require 'ftools'
```

Comment:
If ftools/fileutils is only used to create non-nested directories, then ``Dir.mkdir(string)`` is enough.
No need to require a library that may change again in the future.